war/src/main/webapp/help/parameter/file.html

```diff
@@ -22,6 +22,15 @@
     <code>my.zip</code>
     .) The name will not include the directory name portion.
   </p>
+  <p>
+    Please note that some shells (such as the default /bin/sh) may not be able
+    to recognize environment variables with special symbols like abc.zip. Try
+    using shebang like
+    <code>#!/bin/bash</code>
```

Comment:
`bash` seems to be one of the shells that doesn't support it. What does doing this accomplish?

Reply:
I confirmed `bash` can support it on my Mac OSX and Ubuntu 22.04, and the version is "3.2.57"/"5.0.17"

I use the following command to test it:
```
$ env "abc.zip=foobar" /bin/bash -c "printenv abc.zip"
foobar

$ env "abc.zip=foobar" /bin/sh -c "printenv abc.zip"
$ env "abczip=foobar" /bin/sh -c "printenv abczip"
foobar
```

Reply:
Interesting, I did not expect that to be the result. Thanks for those examples.

Reply:
I assume `dash` behaves the same after a quick look at its sources, so this could be rephrased as "Some shells, such as `dash`, or `/bin/sh` provided by `dash`, do not make such environment variables available to child processes" or similar.

(Also suggesting to replace the "recognize", while that seems to be what's happening in `dash` per `is_in_name`, it reads a bit weird from an end user POV. Thoughts?)

Reply:
Thank you for making it clear!